### DISCUSSION TITLE
Exporting fragment files for custom MACS2 peak calls

### DISCUSSION THREAD

**Discussion by dagarfield** [GreenleafLab/ArchR#1905] [Questions / Documentation]
I know that the proper answer to this question is, "use the reproducible peak calls ArchR does for you already", but I have interest in messing around a bit with MACS peak calling parameters on the signal for each cluster in an ArchR project. 

Do you have advice on the best export method to do this that would be compatible with MACS2? I exported cluster-specific fragments as BED files, but the lack of strand information is causing MACS to complain. Since strand isn't normally a part of the fragment information, presumably you found some way around this for your peak calling?

**Comment by rcorces** [comment]
I think your best bet is to export fragments, convert them to single base insertions (the ends of the fragments) and then use `--shift -75 --extsize 150`. I think if you use `--nomodel` with MACS2 in combo with the shift and extsize params you wont need to worry about the strand?

**Comment by dagarfield** [comment]
I think the strand shouldn't matter, but for whatever reason, I couldn't get it to run without a strand. So I just made them all positive and the peaks look great....yay.

Interestingly, the conversion to single-base also didn't seem to impact things.
